client/components/chart/style.scss

```
@@ -1,15 +1,18 @@
+@import "@wordpress/base-styles/breakpoints";
+
+$barChartBorderRadius: 2px;
+$yearBarChartBorderRadius: 4px;
```

The naming convention in stylesheets is dash-case, so these lines should be:

```suggestion
$bar-chart-border-radius: 2px;
$year-bar-chart-border-radius: 4px;
```